packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -124,10 +124,12 @@ const { densityClass } = useDensity(props);
     font-family: var(--onyx-font-family);
     color: var(--onyx-color-text-icons-neutral-intense);
     text-align: left;
+    contain: paint;
```

Comment:
will this make trouble in the future e.g. with a dropdown opening from within a cell?

Reply:
I think we can only prevent this if we know the exact table height so we can size the column hover effect without `contain: paint` or `overflow: hidden`...but this means we propably need JavaScript...

Reply:
@BoppLi I am not sure if I should take care of this in this PR or if we check this later or we might even place the dropdown absolute idk

Reply:
it's fine for me. let's try to remember it when it comes to it. 👍